### DISCUSSION TITLE
Need Help Understanding Preprocessing Step

### DISCUSSION THREAD

**Discussion by evilc3** [NVIDIA-NeMo/NeMo#4420] [Q&A] [answered]
Hi are we not using normalising using inferencing ? 

        cfg = copy.deepcopy(asr_model.confg)
        OmegaConf.set_struct(cfg.preprocessor, False)
        # some changes for streaming scenario
        cfg.preprocessor.dither = 0.0
        cfg.preprocessor.pad_to = 0
        cfg.preprocessor.normalize = "None"

Also while streaming thought mic  I was normalizing  the audio by dividing it by  audio / (1 << 15) (had seen this is some tutorial) 

Model Used.: ctcconformerbpe  small

**Comment by titu1994** [comment, marked as answer]
During streaming, you should normalize by batch statistics, it usually works fine compared to global normalization.
